drydock_backups/patches/drydock-multipurpose-jobs

```diff
@@ -113,6 +123,14 @@ spec:
                 - name: BACKUP_CUSTOM_STORAGE_ENDPOINT
                   value: '{{ BACKUP_CUSTOM_STORAGE_ENDPOINT }}'
                   {% endif %}
+                {% elif BACKUPS_STORAGE_SERVICE == "azure" %}
+                - name: AZURE_CONTAINER_NAME
+                  value: '{{ AZURE_CONTAINER_NAME }}'
+                - name: AZURE_CONTAINER_SAS_TOKEN
+                  value: '{{ AZURE_CONTAINER_SAS_TOKEN }}'
```

Comment:
This `SAS TOKEN` seems to be the authentication method to add files, is this a static token? i mean, if i define this variable, the token won't expire or for some reason i will need to change it?

Reply:
this token is manually created from the Azure console, where you can also configure its permissions, expiration time, allowed IPs, and some other settings, [here is more information](https://learn.microsoft.com/es-mx/rest/api/storageservices/delegate-access-with-shared-access-signature)